### PR TITLE
First translation instructions

### PR DESCRIPTION
**Pull request by commjoen**
Following the chat on Slack, these are new instructions for setting up a translation.

Thank you for submitting a Pull Request to the Mobile Application Security Verification Standard. Please make sure that:

- [ ] Your PR is linked to an issue.
- [x] Your PR follows the [contribution guidelines](https://github.com/OWASP/owasp-masvs/blob/master/CONTRIBUTING.md "Contribution guidelines")
- [x] All links are valid and use HTTPS and have been formatted as [TEXT](URL “NAME”)
- [ ] In case of a translation: have you made sure that the translation from EN is as close to the EN document as possible?

If your pull-request has a focus of rewriting a requirement and/or adding a new one, please make sure you have a backing issue in which your change has been discussed and agreed. If there is none, then your PR will be closed without further review.

### CODE REVIEW

**Comment by OMTGreleaser** [comment]
Results for commit 7cba2eb542e2ec0127274cc865c21d188a3a7bc7: Broken link result: 0 .
  
 Markdown result errorlength: 0 .

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -49,6 +49,18 @@ See also the following article for further explanation on "[How to Keep a Downst
...
 
+### Start new translation
+
+Translating the MASVS in a new language is great, but given the nature of the MASVS, we have to make sure that the translation can be verified by a second translator who can verify that the English version of the MASVS has been translated properly. So when you want to start a translation: contact us on Slack and together we can make sure that you have your co-translator ready. Once you are all set, we execute the following steps:
+
+1. Follow step 1-8 of [How to set up my contributor environment](##How to set up my contributor environment).
+2. Create a pull request for the langauge you want to support. All you have to do is create a copy of the `Document` folder and check that in, annotated with the language of your choice. For instance, if you want to translate the document into Dutch (NL), then you create a copy of the `Document` folder and check it in as `Document-nl`.
```
Comment (TheDauntless) [inline CONTRIBUTING.md]: "check that in" -> commit it ?

into Dutch (NL), then you  -> into Dutch (NL), you

check it in as -> commit it as

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -49,6 +49,18 @@ See also the following article for further explanation on "[How to Keep a Downst
...
+3. Start the actual translation, one PR after another, which then needs to be reviewed by your fellow translator. Try to do this 1 chapter per PR, so that the reviewer can easily fix it.
```
Comment (TheDauntless) [inline CONTRIBUTING.md]: so that the reviewer can easily fix it. -> so that the reviewer can give feedback on short submissions without being overwhelmed.

**Comment by OMTGreleaser** [comment]
Results for commit df3bbb12bc29580df5023c18f7f7809b5b342b3d: Broken link result: 0 .
  
 Markdown result errorlength: 0 .

**Pull request merged**